src/lib.rs

```diff
@@ -42,6 +42,7 @@
 #[cfg(any(test, feature = "alloc"))]
 #[cfg_attr(test, macro_use)]
 extern crate alloc;
+extern crate core;
```

Comment:
Hmm, not sure this belongs in this commit?

Reply:
Oops, not sure how this slipped in there. Fixed.